love/src/components/AuthList/SummaryAuthList/SummaryAuthList.jsx

```diff
@@ -0,0 +1,483 @@
+            <span>{v}</span>
+            <Hoverable top={true} center={true} inside={true}>
+              <span onClick={() => this.removeIdentity(target, v, type)}>x</span>
+              <div className={styles.removalTooltip}>Request {type} removal</div>
+            </Hoverable>
+          </div>
+        ))}
+      </div>
+    );
+  };
+
+  removeIdentity = (targetCSC, identityToRemove, type) => {
+    const { user } = this.props;
+    const { userIdentity } = this.state;
+    let modalText = '';
+    if (type === 'CSC') {
+      modalText = (
+        <span>
+          You are about to request the removal of <b>{identityToRemove}</b> from <b>{targetCSC}</b> unauthorization
+          list.
+          <br></br>
+          Are you sure?
+        </span>
+      );
+    } else if (type === 'User') {
+      if (userIdentity !== identityToRemove) {
+        modalText = (
+          <span>
+            You are about to request the removal of <b>{identityToRemove}</b> from <b>{targetCSC}</b> authorization list
+            <br></br>
+            Are you sure?
+          </span>
+        );
+      } else {
+        modalText = (
+          <span>
+            You are about to relinquish your authorization.
+            <br></br>
+            <b>This action will be resolved automatically and won't need verification</b>
+            <br></br>
+            Are you sure?
+          </span>
+        );
+      }
+    }
+
+    this.setState({
+      removeIdentityModalShown: true,
+      removeIdentityModalText: modalText,
+    });
+
+    if (type === 'User') {
+      this.setState({
+        removeIdentityRequest: () => {
+          ManagerInterface.requestAuthListAuthorization(user, targetCSC, `-${identityToRemove}`, '').then(() => {
+            this.setState({ removeIdentityModalShown: false });
+          });
+        },
+      });
+    } else if (type === 'CSC') {
+      this.setState({
+        removeIdentityRequest: () => {
+          ManagerInterface.requestAuthListAuthorization(user, targetCSC, '', `-${identityToRemove}`).then(() => {
+            this.setState({ removeIdentityModalShown: false });
+          });
+        },
+      });
+    }
+  };
+
+  restoreToDefault(...args) {
+    const { user, authlistState } = this.props;
+    for (let i = 0; i < args.length; i++) {
+      const selectedCSC = args[i];
+      const authlist = Object.entries(authlistState).find(([key]) => {
+        const keyTokens = key.split('-');
+        const csc = `${keyTokens[1]}:${keyTokens[2]}`;
+        return csc === selectedCSC;
+      })[1];
+
+      const cscsToChange = selectedCSC;
+      const authorizedUsers = authlist.authorizedUsers
+        .split(',')
+        .map((x) => `-${x}`)
+        .join(',');
+      const nonAuthorizedCSCs = authlist.nonAuthorizedCSCs
+        .split(',')
+        .map((x) => `-${x}`)
+        .join(',');
+      console.log('Request: ', cscsToChange, authorizedUsers, nonAuthorizedCSCs);
```

Comment:
Is this console.log must be replaced by the next line?

Reply:
I've removed this on further commits. Thanks!